src/components/ButtonPanel.js

```diff
@@ -1,40 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import '../index.css';
 import Button from './Button';
 
-const ButtonPanel = () => (
+const ButtonPanel = ({ onClick }) => (
   <>
     <div>
-      <Button name="AC" />
-      <Button name="+/-" />
-      <Button name="%" />
-      <Button name="÷" />
+      <Button name="AC" handleOnClick={onClick} />
+      <Button name="+/-" handleOnClick={onClick} />
+      <Button name="%" handleOnClick={onClick} />
+      <Button name="÷" handleOnClick={onClick} />
```

Comment:
You need to  implement <code>handleClick</code> (in your case, onClick) function (<a href="https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png">screenshot</a>). e.g. 

```JavaScript
handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
};
```
Then, each button in buttonPanel will become something like:

`<Button name="AC" handleOnClick={this.handleClick} />`

Note: Since your `ButtonPanel` is implemented as a functional component, pls convert the function above to its functional component equivalent.